internal/agent/harness/sufficiency.go

```
@@ -19,51 +19,207 @@ package harness
+	// This keeps number-adjacent Chinese evidence verifiable without spaCy.
+	if isCJK(text) {
+		return extractCJKSubstrings(text)
+	}
+	return nil
+}
+
+// extractCJKSubstrings returns runs of CJK characters as entity candidates.
+// This is intentionally coarse (no NER): it only preserves the substring-match
+// path for CJK evidence that the union matcher below can still use.
+func extractCJKSubstrings(text string) []string {
+	re := regexp.MustCompile(`[\u4e00-\u9fff]+`)
```

golangci-lint (2.12.2)

[error] SA1000: error parsing regexp: invalid escape sequence: `\u`

(staticcheck)